### PR TITLE
Find profiles

### PR DESCRIPTION
**Pull request by alex-setyawan**

### CODE REVIEW

**Comment by jovantanyk** [comment]
Todo:
- [ ] Update BloodType.java 
- [ ] Update Incorrect fields in JsonAdaptedPerson
- [ ] PersonBuilder returns Null Pointer on invalid field

**Review by jovantanyk — APPROVED** [review]
Updated BloodType.java and Test Cases

File: src/main/java/seedu/address/model/person/BloodType.java [hunk 1]
```diff
@@ -3,64 +3,56 @@
...
-    public String getRh() {
-        return this.rh == Rh.POSITIVE ? "+" : "-";
+    /**
+     * Returns given placeholder string if value field is not initialised
+     *
+     * @param alt An alternate return value
+     * @return placeholder string
+     */
+    public String orElse(String alt) {
+        return this.bloodType == null ? alt : this.toString();
```
Comment (laney0808) [inline src/main/java/seedu/address/model/person/BloodType.java:45]: return this.bloodType == null ? alt : this.bloodType; *to be consistent

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 2]
```diff
@@ -76,16 +78,18 @@ public void execute_multipleKeywords_multiplePersonsFound() {
 
     @Test
     public void toStringMethod() {
-        NameContainsKeywordsPredicate predicate = new NameContainsKeywordsPredicate(Arrays.asList("keyword"));
+        NameContainsKeywordsPredicate predicate = new NameContainsKeywordsPredicate(List.of("keyword"));
```
Comment (laney0808) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java:81]: why ah

**Review by laney0808 — APPROVED** [review]
Ok. Let's proceed with the test cases. Bugs may be found during tests so more edition may be expected.

**Pull request merged**